src/app/modules/bounce/bounce.service.ts

```diff
@@ -101,20 +108,21 @@ export const isValidSnsRequest = async (
 // Writes a log message if all recipients have bounced
 const logCriticalBounce = (
   bounceDoc: IBounceSchema,
-  formId: string,
   notification: IEmailNotification,
 ): void => {
-  if (
-    !bounceDoc.hasAlarmed &&
-    bounceDoc.bounces.every((emailInfo) => emailInfo.hasBounced)
-  ) {
+  if (bounceDoc.bounces.every((emailInfo) => emailInfo.hasBounced)) {
     logger.warn({
-      type: 'CRITICAL BOUNCE',
-      formId,
-      recipients: bounceDoc.bounces.map((emailInfo) => emailInfo.email),
-      // We know for sure that critical bounces can only happen because of bounce
-      // notifications, so this casting is okay
-      bounceInfo: (notification as IBounceNotification).bounce,
+      message: 'CRITICAL BOUNCE',
+      meta: {
+        action: 'Received critical bounce',
```

Comment:
shouldn't action be `isValidSnsRequest`? We should follow a convention haha. Maybe the message can be "Received critical bounce" and add "CRITICAL BOUNCE" as `meta.bounceType` or something